be/src/env/s3_client.h

```diff
@@ -0,0 +1,90 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#pragma once
+
+#include <aws/s3/S3Client.h>
+#include <aws/transfer/TransferManager.h>
+
+#include <memory>
+
+#include "env/cloud_storage_client.h"
+
+namespace starrocks {
+
+class S3ClientConfiguration {
+public:
+    std::string access_key_id = "";
```

Comment:
I suggest not to wrap this client configuration, because there are many fields that want to be set by users, eg. `endpointOverride'. 

Wrap `access_key_id` and `secret_access_key` is ok.